From d63ce44ced1cf8212bdf726eb8827f2f5c3b2b82 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:37:58 -0400
Subject: [alsa] hda: Remove unused variable

Message-id: <140972823573-375-patchset-jkysela@redhat.com>
Patchwork-id: 91071
O-Subject: [RHEL 7.1 PATCH 374/442] ALSA: hda: Remove unused variable
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

'status' is not used in the function. Remove it.

Signed-off-by: Sachin Kamat <sachin.kamat@samsung.com>
Tested-by: Dylan Reid <dgreid@chromium.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: d5471e67229adb31a1e5f026955d006f06315f3d)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_tegra.c b/sound/pci/hda/hda_tegra.c
index cc4a923..dc4bb21 100644
--- a/sound/pci/hda/hda_tegra.c
+++ b/sound/pci/hda/hda_tegra.c
@@ -271,13 +271,9 @@ static int hda_tegra_resume(struct device *dev)
  struct snd_card *card = dev_get_drvdata(dev);
  struct azx *chip = card->private_data;
  struct hda_tegra *hda = container_of(chip, struct hda_tegra, chip);
- int status;
 
  hda_tegra_enable_clocks(hda);
 
- /* Read STATESTS before controller reset */
- status = azx_readw(chip, STATESTS);
-
  hda_tegra_init(hda);
 
  azx_init_chip(chip, 1);
-- 
1.7.1